styles.css

```
@@ -314,27 +314,41 @@ body {
+  height: 390px;
   background: linear-gradient(100deg,
       #f0f0f0 30%,
       #e0e0e0 40%,
       #f0f0f0 60%);
   background-size: 400% 100%;
   animation: shimmer 1.5s infinite linear;
-  display: flex;
   flex-direction: column;
   background-color: #fff;
   border-radius: 16px;
+  width: 100%;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)